var/spack/repos/builtin/packages/hp2p/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Hp2p(AutotoolsPackage):
+    """HP2P (Heavy Peer To Peer) benchmark is a test which performs MPI
+    Point-to-Point non-blocking communications between all MPI processes. Its
+    goal is to measure the bandwidths and the latencies in a situation where
+    the network is busy. This benchmark can help to detect network problems
+    like congestions or problems with switches or links.
+    """
+
+    home = "https://github.com/cea-hpc/hp2p"
```

Comment:
This needs to be the `homepage` property:

```suggestion
    homepage = "https://github.com/cea-hpc/hp2p"
```

Reply:
Confirmed the  homepage exists (when specified using `homepage`).